src/components/PizzaCard.vue

```diff
@@ -15,19 +15,24 @@ const pizza = computed<Pizza | undefined>(() => pizzaList.value[props.id]);
 
 </script>
 <template>
-  <div class="grid place-items-center bg-cyan-900 shadow-lg">
+  <div class="grid bg-cyan-900 shadow-lg">
     <img
       :alt="`Image de ${pizza?.name}`"
       :src="pizza?.image"
       class="max-w-screen aspect-video h-48 w-full text-clip object-cover"
     />
-    <div class="w-3/4 border-b-2">
-      <p class="text-center text-2xl font-bold">
-        {{ pizza?.name }}
-      </p>
+    <div class="flex h-44 w-full flex-col items-center p-2">
+      <div class="mb-2 w-3/4 border-b-2">
+        <p class="text-center text-2xl font-bold">
+          {{ pizza?.name }}
+        </p>
+      </div>
+      <div class="flex items-center grow">
```

Comment:
Because of the fixed height and the "grow" there is a lot of padding and this look odd.
![image](https://github.com/user-attachments/assets/129c6bd7-83be-4ffa-8f1b-10abcdf9266c)

I think that will look better without the "h-44" and the "grow" without changing the better image handling.
![image](https://github.com/user-attachments/assets/d4ca9885-3af7-43b0-b6cc-ff78e201409b)

What do you think about it ?

Reply:
The issus i had was that all the card doesn't have the same height without this, pizza with a lot of ingredients like the Savoyarde are too big.

Reply:
With what I suggested It may look better and the height stay the same as other PizzaCard on the same grid line. In addition, when there are too many ingredients (which may not occurs but :shrug:), there is an overflow with your code.

![image](https://github.com/user-attachments/assets/7bc52ba9-40c7-443e-9dff-46048cde6d42)

You can try this code instead:
```html
    <div class="flex w-full flex-col items-center p-2">
      <div class="mb-2 w-3/4 border-b-2">
        <p class="text-center text-2xl font-bold">
          {{ pizza?.name }}
        </p>
      </div>
      <div class="flex">
        <p class="text-center">
          <span class="font-black">Ingrédients :</span> {{ pizza?.ingredients.join(', ') }}<br>
          <span class="font-black">Allergènes :</span> {{ pizza?.allergens.join(', ') }}
        </p>
      </div>
    </div>
```
Which looks like that : 
![image](https://github.com/user-attachments/assets/68beba87-da24-48d3-95cd-a6b557e3a05d)